lib/puppet/vendor/semantic/lib/semantic.rb

```diff
@@ -0,0 +1,5 @@
+require 'puppet/vendor/semantic_puppet/lib/semantic_puppet'
```

Comment:
the garethr/puppet-module-skeleton does `require 'puppet/vendor/semantic/lib/semantic'` directly without actually requiring puppet itself, so we don't have the vendor machinery or load paths manipulated to just do a `require 'semantic_puppet'`. Hence spelling out the path fully. https://github.com/garethr/puppet-module-skeleton/blob/master/skeleton/Rakefile#L5-L14